src/components/app-header/app-header.component.tsx

```diff
@@ -1,79 +1,52 @@
 import React from 'react';
 import { compose, withProps } from 'recompose';
+import Header from '@fellesdatakatalog/internal-header';
 
 import localization from '../../services/localization';
 import './app-header.scss';
 import { authService } from '../../services/auth/auth-service';
+import { ResourceRole, User } from '../../lib/auth/auth';
+import { getUserThemeProfile } from '../../lib/theme-profile';
 import { getConfig } from '../../config';
-import { User } from '../../lib/auth/auth';
 
 interface Props {
   user: User;
+  resourceRoles: ResourceRole[];
 }
 
-export const AppHeaderPure = ({ user }: Props) => {
-  const logoImagePath = getConfig().useDemoLogo
-    ? '/img/logo-registrering-demo.svg'
-    : '/img/logo-registrering.svg';
+export const AppHeaderPure = ({ user, resourceRoles }: Props) => (
+  <div>
```

Comment:
Her kan vi bruker fragment (<></>) i stedet for div

Reply:
Forenklet uten fragment